src/Views/Dashboard/Dashboard.jsx

```diff
@@ -104,6 +108,12 @@ const Dashboard = () => {
     }
   };
 
+  useEffect(() => {
+    if (isMobile) {
+      setViewState("board");
+    }
+  }, [isMobile]);
+
```

Comment:
```suggestion
  useEffect(() => {
    if (isMobile && viewState !== "board") {
      setViewState("board");
    }
  }, [isMobile]);

```

you could get stuck in an infinite loop if you don't check the viewState also